types/miniprogram/index.d.ts

```diff
@@ -0,0 +1,1936 @@
+     */
+    type?: string;
+
+    /**
+     * Displaying duration, in ms, 2000 by default.
+     */
+    duration?: number;
+}
+
+type ChoosePhoneContactArgs = AsyncCallback<{
+    name: string;
+    mobile: string;
+}>;
+
+interface CreateAnimationArgs {
+    /**
+     * Animation duration, in ms, 400 by default.
+     */
+    duration?: number;
+
+    /**
+     * Define animation effect, linear by default, effective
+     *  values including linear, ease, ease-in, ease-in-out,
+     * ease-out, step-start and step-end .
+     */
+    timeFunction?: string;
+
+    /**
+     * Animation delay, in ms, 0 by default.
+     */
+    delay?: number;
+
+    /**
+     * Set transform-origin, 50% 50% 0 by default.
+     */
+    transformOrigin?: string;
+}
+
+interface Animation {
+    /**
+     * Transparency, range 0~1.
+     */
+    opacity: (value: number) => Animation;
+
+    /**
+     * Color value.
+     */
+    backgroundColor: (color: string) => Animation;
+
+    /**
+     * Set the width:length values, in px, such as 300 px.
+     */
+    width: (value: number) => Animation;
+
+    /**
+     * Set the height:length values, in px, such as 300 px.
+     */
+    height: (value: number) => Animation;
+
+    /**
+     * Set the top:length values, in px, such as 300 px.
+     */
+    top: (value: number) => Animation;
+
+    /**
+     * Set the left:length values, in px, such as 300 px.
+     */
+    left: (value: number) => Animation;
+
+    /**
+     * Set the bottom:length values, in px, such as 300 px.
+     */
+    bottom: (value: number) => Animation;
+
+    /**
+     * Set the right:length values, in px, such as 300 px.
+     */
+    right: (value: number) => Animation;
+
+    /**
+     * deg range -180 ~ 180, rotate by deg degrees clockwise from origin.
+     */
+    rotate: (value: number) => Animation;
+
+    /**
+     * deg range -180 ~ 180, rotate by deg degrees on X axis.
+     */
+    rotateX: (value: number) => Animation;
+
+    /**
+     * deg range -180 ~ 180, rotate by deg degrees on Y axis.
+     */
+    rotateY: (value: number) => Animation;
+
+    /**
+     * deg range -180 ~ 180, rotate by deg degrees on Z axis.
+     */
+    rotateZ: (value: number) => Animation;
+
+    /**
+     * Same as [transform-function rotate3d](https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function/rotate3d).
+     */
+    rotate3d: (x: number, y: number, z: number) => Animation;
+
+    /**
+     * When there is only one parameter, it indicates scaling sx
+     * times on X and Y axises at the same time.
+     *
+     * When there are two parameters, it indicates scaling sx times
+     * on X axis and sy times on Y axis.
+     */
+    scale: (sx: number, sy?: number) => Animation;
+
+    /**
+     * Scale sx times on X axis.
+     */
+    scaleX: (sx: number) => Animation;
+
+    /**
+     * Scale sy times on Y axis.
+     */
+    scaleY: (sy: number) => Animation;
+
+    /**
+     * 	Scale sz times on Z axis.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/miniprogram/index.d.ts;
  Line: 326;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.